### PR TITLE
Ports Different Variants of Nian Plushies and Introduces a Plush Spawner

### PR DESCRIPTION
**Pull request by haydenredacted**
## What Does This PR Do

Ports over the different variants of moth plushies made by pigeonverde from beestation.
Adds a new type of spawner that only includes plushies and adds it to the maintenance loot pool
Due to more things being added into the maintenance loot pool, tools and materials have been made more rare.

## Why It's Good For The Game

Who doesn't love more plushies especially cute nian ones
More varied maintenance loot also helps spice up each round and more unique spawners helps make mapmaking more fun 

## Images of changes

moth plushies

## Testing

Spawned all the different types of nian plushies.
Saw all the types of plushies in maintenance 

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
add: Adds more nian plush variants
add: New plushie spawner
tweak: tools and materials spawning in maintenance is more rare
/:cl:

### CODE REVIEW

**Comment by Komrad822** [comment]
The plush wars have begun

**Comment by haydenredacted** [comment]
> Were you going to kill off `/obj/random/carp_plushie` and `/obj/random/plushie` in this PR or should I take care of it after this PR goes in?

You can take care of it please.

**Review by Burzah — APPROVED** [review]

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -879,7 +879,109 @@
 	visible_message("<span class='danger'>Buzzzz!</span>")
 	cooldown = TRUE
 	addtimer(VARSET_CALLBACK(src, cooldown, FALSE), 3 SECONDS)
-
+	
+/obj/item/toy/plushie/nianplushie/monarch // the beginning of the nian plush army
```
Comment (Contrabang) [inline code/game/objects/items/toys.dm:883]: This comment doesnt add any useful information, remove it please
```suggestion
/obj/item/toy/plushie/nianplushie/monarch
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/_globalvars/lists/maint_loot_tables.dm [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 	list(
 		// Tools
-		/obj/effect/spawner/random/engineering/tools,
+		/obj/effect/spawner/random/engineering/tools = 3,
 
 		// Materials
-		/obj/effect/spawner/random/engineering/materials,
+		/obj/effect/spawner/random/engineering/materials = 3,
+		// Plushies
+		/obj/effect/spawner/random/plushies = 1,
```
Comment (Contrabang) [inline code/_globalvars/lists/maint_loot_tables.dm:9]: Also, please mention in the PR description and changelog that you are making specifically tools and materials more rare

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/_globalvars/lists/maint_loot_tables.dm (hunk 2 above)
Comment (haydenredacted) [reply code/_globalvars/lists/maint_loot_tables.dm:9]: done.

Comment (warriorstar-orion) [reply code/_globalvars/lists/maint_loot_tables.dm:9]: @haydenredacted  I would honestly bump these numbers up a bit, this is an over 10% reduction in those spawns. Could you weight them to 4? We can check the stats once this goes in and make sure everything's sane, and tweak them later.

Comment (haydenredacted) [reply code/_globalvars/lists/maint_loot_tables.dm]: also done.

**Review by warriorstar-orion — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]

**Pull request merged**